processing/text_processing.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+import pandas as pd
+import re
+import string
+import nltk
+from nltk.tokenize import word_tokenize
+from nltk.stem import WordNetLemmatizer 
+
+from nltk.corpus import stopwords
+nltk.download('stopwords')
+stop = stopwords.words('english')
+
+# project_dir = r'D:\GitHub\Projects\Analysis_of_Delivery_Companies_Reviews'
+# os.chdir(project_dir)
+
+def trimSpace(text) :

```

Comment:
Can we remove this, as its a function that exists already so we do not need to create a separate one for the same task :)